src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -131,28 +131,29 @@ limitations under the License.
 
     <event side="server" code="0x00" priority="info" name="StateChanged" optional="true">
       <description>If supported, this event SHALL be generated when there is a change in any of the supported attributes of the Media Playback cluster.</description>
-      <field id="0" name="CurrentState" type="PlaybackStateEnum"/>
-      <field id="1" name="StartTime" type="EPOCH_US"/>
-      <field id="2" name="Duration" type="INT64U"/>
-      <field id="3" name="SampledPosition" type="PlaybackPositionStruct"/>
-      <field id="4" name="PlaybackSpeed" type="single"/>
-      <field id="5" name="SeekRangeEnd" type="INT64U"/>
-      <field id="6" name="SeekRangeStart" type="INT64U"/>
-      <field id="7" name="Data" type="OCTET_STRING" optional="true" length="900" />
-      <field id="8" name="AudioAdvanceUnmuted" type="boolean" default="true"/>
+      <field id="0" name="CurrentState" type="PlaybackStateEnum" min="0x00" max="0x03"/>
+      <field id="1" name="StartTime" type="epoch_us" optional="true"/>
+      <field id="2" name="Duration" type="int64u" optional="true"/>
+      <field id="3" name="SampledPosition" type="PlaybackPositionStruct" optional="true"/>
+      <field id="4" name="PlaybackSpeed" type="single" optional="true"/>
+      <field id="5" name="SeekRangeEnd" type="int64u" optional="true"/>
+      <field id="6" name="SeekRangeStart" type="int64u" optional="true"/>
+      <field id="7" name="Data" type="octet_string" optional="true" length="900"/>
+      <field id="8" name="AudioAdvanceUnmuted" type="boolean" default="0" optional="true"/>
     </event>
   </cluster>
 
   <struct name="TrackStruct">
     <cluster code="0x0506"/>
-    <item name="ID" type="CHAR_STRING" optional="false" length="32"/>
-    <item name="TrackAttributes" type="TrackAttributesStruct" isNullable="true" optional="false"/>
+    <item fieldId="0" name="ID" type="char_string" length="32"/>
+    <item fieldId="1" name="TrackAttributes" type="TrackAttributesStruct"/>
   </struct>
 
   <struct name="TrackAttributesStruct">
     <cluster code="0x0506"/>
-    <item name="LanguageCode" type="CHAR_STRING" optional="false" length="32"/>
-    <item name="DisplayName"  type="CHAR_STRING" isNullable="true" optional="true"/>
+    <item fieldId="0" name="LanguageCode" type="char_string" length="32"/>
+    <item fieldId="1" name="Characteristics" type="CharacteristicEnum" isNullable="true" optional="true" array="true"/>
+    <item fieldId="2" name="DisplayName" type="long_char_string" isNullable="true" optional="true" length="256"/>
     <!-- <item name="Characteristics" type="CharacteristicEnum" optional="true" array="true"/> -->
```

Comment:
Can this be removed now? I think it wasn't able to compile before

Reply:
If it's not compiling and you don't want it, we should remove it from the spec.